lib/web/apiserver_test.go

```diff
@@ -3965,6 +3967,137 @@ func TestChangeUserAuthentication_WithPrivacyPolicyEnabledError(t *testing.T) {
 	require.True(t, apiRes.PrivateKeyPolicyEnabled)
 }
 
+func TestChangeUserAuthentication_settingDefaultClusterAuthPreference(t *testing.T) {
+
+	tt := []struct {
+		cloud               bool
+		name                string
+		numberOfUsers       int
+		resultConnectorName string
+	}{{
+		cloud:               true,
+		name:                "cloud, 1 user; signing with passwordless should change connectorName to `passwordless`",
```

Comment:
```suggestion
		name:                "first cloud sign-in changes connector to `passwordless`",
```

(and move to the line above, as mentioned before.)